From 424c9a32be28be53f525570a7de668efef974f13 Mon Sep 17 00:00:00 2001
From: Prasanth Jayachandran <prasanthj@apache.org>
Date: Mon, 12 Sep 2016 12:09:45 -0700
Subject: [PATCH 0961/1164] CDH-48190: HIVE-14681: Remove AssertionError for
 non zero return codes in QTestUtil cleanup
 (Siddharth Seth reviewed by Hari Subramaniyan)

(cherry picked from commit 518789df22310b2ff95e92d0ee7cf7636daa70b3)

Change-Id: I13a82bd4dbda6a3f89dfea89dbbbc9fbe1368463
---
 .../java/org/apache/hadoop/hive/ql/QTestUtil.java  |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/itests/util/src/main/java/org/apache/hadoop/hive/ql/QTestUtil.java b/itests/util/src/main/java/org/apache/hadoop/hive/ql/QTestUtil.java
index 65f35d0..1bd471b 100644
--- a/itests/util/src/main/java/org/apache/hadoop/hive/ql/QTestUtil.java
+++ b/itests/util/src/main/java/org/apache/hadoop/hive/ql/QTestUtil.java
@@ -926,7 +926,8 @@ public void cleanUp() throws Exception {
       SessionState.get().getConf().setBoolean("hive.test.shutdown.phase", true);
       int result = cliDriver.processLine(cleanupCommands);
       if (result != 0) {
-        Assert.fail("Failed during cleanup processLine with code=" + result);
+        LOG.error("Failed during cleanup processLine with code=" + result + ". Ignoring");
+        // TODO Convert this to an Assert.fail once HIVE-14682 is fixed
       }
       SessionState.get().getConf().setBoolean("hive.test.shutdown.phase", false);
     } else {
-- 
1.7.9.5

